mitodl/hugo-course-publisher
# Update course result content

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #227 
Closes #230

#### What's this PR do?
Adds additional info to each course search result (course number, level, instructors) and tweaks the styling to match [the design](https://projects.invisionapp.com/share/TYYM755E9FJ#/screens/430328458_OCW_Course_Search_Learning_Landing_Page_Courses09-01-2020_No_Facets).

#### How should this be manually tested?
Go to the search page and try out some different searches for courses and resources.  The page should look about the same as the design in terms of what's mentioned in the issues, and the course title/thumbnail should load the course URL when clicked in the same tab.

#### Any background context you want to provide?
(Optional)

#### Screenshots
<img width="1133" alt="Screen Shot 2020-09-30 at 3 34 15 PM" src="https://user-images.githubusercontent.com/187676/94731253-7ddfd000-0332-11eb-8c3e-9e5901f6cdde.png">
